test/e2e/config/fullfunctional.conf.js

```diff
@@ -0,0 +1,118 @@
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const minimist = require('minimist');
+
+chai.use(chaiAsPromised);
+
+const argv = minimist(process.argv.slice(2));
+
+//const specFilesFilter = ['../features/**/*.feature'];
+
+// module.exports = {
+//   chai: chai,
+//   chaiAsPromised: chaiAsPromised,
+//   minimist: minimist,
+//   argv: argv,
+//   specFilesFilter: specFilesFilter
+// }
+//
+
+const jenkinsConfig = [
+
+    {
+        browserName: 'chrome',
+        acceptInsecureCerts: true,
+        nogui: true,
+        chromeOptions: { args: ['--headless', '--no-sandbox', '--disable-dev-shm-usage', '--disable-setuid-sandbox', '--no-zygote ', '--disableChecks'] }
+    }
+];
+
+const localConfig = [
+    {
+        browserName: 'chrome',
+        acceptInsecureCerts: true,
+        chromeOptions: { args: ['--headless1', '--no-sandbox', '--disable-dev-shm-usage', '--disable-setuid-sandbox', '--no-zygote '] },
+        proxy: {
+            proxyType: 'manual',
+            httpProxy: 'proxyout.reform.hmcts.net:8080',
+            sslProxy: 'proxyout.reform.hmcts.net:8080',
+            noProxy: 'localhost:3000'
+        }
+    }
+];
+
+const cap = (argv.local) ? localConfig : jenkinsConfig;
+
+const config = {
+    framework: 'custom',
+    frameworkPath: require.resolve('protractor-cucumber-framework'),
+    specs: ['../features/**/*.feature'],
+    // specs: [
+    //   '../features/**/caseFile.feature',
+    //   '../features/**/login.feature',
+    //   '../features/**/makeDecision.feature',
+    //   '../features/**/parties.feature',
+    //   '../features/**/questions.feature',
+    //   '../features/**/recentEvents.feature',
+    // ],
+    baseUrl: process.env.TEST_URL || 'http://localhost:3000',
+    params: {
+        serverUrls: process.env.TEST_URL || 'http://localhost:3000',
+        targetEnv: argv.env || 'local',
+        // username: process.env.TEST_EMAIL,
+        // password: process.env.TEST_PASSWORD,
+        username: 'vmuniganti@mailnesia.com',
```

Comment:
again might be test users but I don't think we should have any credentials stored in the repo